docs/user-guide/influxdbsink.rst

```diff
@@ -0,0 +1,65 @@
+.. _influxdbsink:
+
+#############
+InfluxDB Sink
+#############
+
+The InfluxDB Sink connector is a Kafka connector that allows data to be written from Kafka topics to InfluxDB.
+
+Tags
+====
+
+In InfluxDB, tags are a type of metadata used to organize and filter data.
+Tags are optional but they provide additional context when querying data.
+
+From the ``lsst.example.skyFluxMetric`` metric payload:
+
+.. code:: json
+
+    {
+        "timestamp": 1681248783000000,
+        "band": "y",
+        "instrument": "LSSTCam-imSim",
+        "meanSky": -213.75839364883444,
+        "stdevSky": 2328.906118708811,
+    }
+
+``band`` and ``instrument`` are good candidates for tags.
+
+When you query data from InfluxDB, you can use tags to filter the data and aggregate it in different ways.
+
+For example, you might query the ``lsst.example.skyFluxMetric`` metric and group the results by the ``band`` tag to see ``meanSky`` and ``stdevSky`` in each band.
+Or you might filter the data to only show values for a specific band or instrument.
+
```

Comment:
You mentioned that using `id` as the tag is not a good idea because of the high cardinality.  That might be a good info to have here too.